File: C/Exercicio06.c
/*
Tendo como dados de entrada a altura de uma pessoa, construa um algoritimo
que calcule seu peso ideal, usando a seguinte formula:
    (72,7 * altura) - 58
*/

#include<stdio.h>

int main(){
	
	float altura, peso_ideal;
	
	printf("Informe sua altura:");
	scanf("%f", &altura);
	
	peso_ideal = (72.7 * altura) - 58;
	
	printf("Seu peso ideal e: %.2f", peso_ideal);
}

File: Java/Exercicio06.java
/*
Tendo como dados de entrada a altura de uma pessoa, construa um algoritimo
que calcule seu peso ideal, usando a seguinte formula:
    (72,7 * altura) - 58
*/

package Java;

import java.util.Scanner;

public class Exercicio06{
	
	public static void main(String[] args) {
		
		System.out.println("Informe a sua altura:");
		float altura = new Scanner(System.in).nextFloat();
		
		double pesoIdeal = (72.7 * altura) - 58;
		
		System.out.println("Seu peso ideal e: " + String.format("%.2f", pesoIdeal));
	}
}